test/test-project/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+#required by GitHub Actions
+sudo chown -R 1001:121 "/root/.npm"
+cd app
+npm install
+cd test/test-project
+npm install
+npm run test:project
+#installing html template globally before global tests
+PUPPETEER_SKIP_DOWNLOAD=true npm install -g @asyncapi/html-template@0.16.0
+#remove previously installed html template to make sure it is not picked up in the test
+rm -rf node_modules/@asyncapi/html-template
+rm -rf ../../node_modules/@asyncapi/html-template
+npm run test:global
```

Comment:
Since we are using Docker-compose wouldn't it be fitting to wrap this in a dockerfile instead 🤔

Reply:
@jonaslagoni  You can also run it locally as I understand from new section in docs :)

Reply:
sure we can use Dockerfile and instead of `docker-compose up` we could do `docker build`

I choose docker compose:
- because I kinda prefer to read docker compose config file instead of Dockerfile
- in `docker-compose` stored under `test-project` I could map volumes like `../../:/app` and in Dockerfile it would not work because of context related issues and Dockerfile would have to be on the root of the project. I felt that this is just weird that all `test-project` related files are in the related directory, except of this one single Dockerfile

Reply:
Yea, I guess, the only reason why I don't like the script (even if it should be run locally) is that it does not force a clean environment but I see your point 👍